src/server-extension/resolvers/VideosResolver/index.ts

```diff
@@ -194,6 +198,61 @@ export class VideosResolver {
     return result as VideosConnection
   }
 
+  @Query(() => [VideoReturnType])
+  async dumbPublicFeedVideos(
+    @Args() args: DumbPublicFeedArgs,
+    @Info() info: GraphQLResolveInfo,
+    @Ctx() ctx: Context
+  ): Promise<VideoReturnType[]> {
+    const tree = getResolveTree(info)
+
+    const sqlArgs = parseSqlArguments(model, 'Video', {
+      limit: args.limit,
+      where: args.where,
+    })
+
+    const videoFields = parseAnyTree(model, 'Video', info.schema, tree)
+
+    const listQuery = new ListQuery(model, ctx.openreader.dialect, 'Video', videoFields, sqlArgs)
+
+    let listQuerySql = listQuery.sql
+
+    listQuerySql = extendClause(
+      listQuerySql,
+      'WHERE',
+      `"video"."include_in_home_feed" = true AND "video"."id" NOT IN (${args.skipVideoIds
+        .map((id) => `'${id}'`)
+        .join(', ')})`,
+      'AND'
+    )
+
+    listQuerySql = extendClause(listQuerySql, 'ORDER BY', 'RANDOM()', '')
+    ;(listQuery as { sql: string }).sql = listQuerySql
+
+    const result = await ctx.openreader.executeQuery(listQuery)
+
+    return result as VideoReturnType[]
+  }
+
+  @Mutation(() => SetPublicFeedResult)
+  @UseMiddleware(OperatorOnly(OperatorPermission.SET_PUBLIC_FEED_VIDEOS))
+  async setPublicFeedVideos(@Args() { videoIds }: SetPublicFeedArgs): Promise<SetPublicFeedResult> {
+    const em = await this.em()
+
+    return withHiddenEntities(em, async () => {
+      const result = await em
+        .createQueryBuilder()
+        .update<Video>(Video)
+        .set({ includeInHomeFeed: true })
```

Comment:
How to remove video from home feed?

Reply:
Addressed in [4e77fc0](https://github.com/Joystream/orion/pull/304/commits/4e77fc062df80011ee1ac71885943701a405f5ee)